docs/release_notes.md

```diff
@@ -85,7 +85,7 @@ Python dependency versions.
 Minor bug fixes were made in each of the repositories. For full change lists, see each
 repository’s CHANGELOG.md file.
 
-The known issue in which the `sat` command was not available by default in `sat bash` was fixed.
+The known issues listed under the SAT 2.2 release were fixed.
```

Comment:
nit: could link to the section describing those issues

Reply:
I added a link here.